source/index.html.md

```diff
@@ -2247,27 +2247,28 @@ Hence, we recommend calling `GetPendingTxn` around 1-2 transaction epochs after
 
 ### STATUS CODES
 
-From Zilliqa `V6.3.0` onwards
+From Zilliqa `V7.0.0` onwards
 
 **Confirmed Transactions**
 
-| `code` | Transaction Status          |
-| ------ | --------------------------- |
-| 0      | Transaction is not pending  |
+| `code` | Transaction Status            |
+| ------ | ----------------------------- |
+| 3      | Transaction is confirmed      |
+| 2      | Transaction is soft-confirmed |
 
 **Pending Transactions**
 
 | `code` | Transaction Status             |
 | ------ | ------------------------------ |
-| 1      | Nonce is higher than expected  |
-| 2      | Microblock gas limit exceeded  |
-| 3      | Consensus failure in network   |
+| 4      | Nonce is higher than expected  |
+| 5      | Microblock gas limit exceeded  |
+| 6      | Consensus failure in network   |
 
-**Unknown Transactions**
+**Dispatched Transactions**
 
-| `code` | Transaction Status     |
-| ------ | ---------------------- |
-| 4      | Transaction not found  |
+| `code` | Transaction Status                      |
+| ------ | --------------------------------------- |
+| 1      | Transaction was recieved by the lookup  |
```

Comment:
```suggestion
| 1      | Transaction was received by the lookup  |
```

Reply:
@kaikawaliu  need to ensure we don't have similar typo on mongo side

Reply:
Did u mean JS side ?

Reply:
yes